build/schemas/note/message.json

```diff
@@ -25,7 +25,13 @@
       "required": [
         "content"
       ],
-      "description": "Message represents the minimum possible contents for a GoBL document type."
+      "description": "Message represents a simple message object with a title and some content meant.",
+      "examples": [
+        {
+          "content": "This is an example message.",
+          "title": "Example Title"
+        }
+      ]
```

Comment:
It's great to have examples! Personally, I like them to look as real as possible. I hate it when I find examples like:` { "first_name": "first name", "last_name": "last name", "vat_code": "99999" }`. Those add little value. A real-looking example provides information about the meaning of the field implicitly, helping readers reinforce their understanding of the semantics.